devel/restbed/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           openssl 1.0
+
+github.setup        Corvusoft restbed 4.8
+revision            0
+categories          devel
+license             AGPL-3
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Framework that brings asynchronous RESTful functionality to C++14 applications.
+long_description    Restbed is a comprehensive and consistent programming model for building applications \
+                    that require seamless and secure communication over HTTP, with the ability to model \
+                    a range of business processes, designed to target mobile, tablet, desktop \
+                    and embedded production environments.
+homepage            https://www.corvusoft.co.uk
+checksums           rmd160  45f056be2a4f79f51b95c035bcf594468ff5bec7 \
+                    sha256  8d2959b8a4b14a5c4f15364220b9b4be203b9af9ba46c439fa68084e2910f27f \
+                    size    145776
+
+depends_lib-append  port:asio
+
+patchfiles          patch-CMakeLists.diff
+
+compiler.cxx_standard 2014
+
+configure.args-append \
+                    -DBUILD_IPC=OFF \
+                    -DBUILD_SSL=OFF \
+                    -DBUILD_TESTS=OFF
+
+pre-destroot {
+    xinstall -d ${destroot}${prefix}/include/${name}
+}
+
+post-destroot {
+    # Fix permissions; setting to 0644 does not have a needed effect.
+    file attributes ${destroot}${prefix}/include/${name} -permissions 0755
```

Comment:
Maybe rewrite the pre-destroot to the following:

```tcl
xinstall -d -m 0755 ${destroot}${prefix}/include/${name}
```

Reply:
Should probably work, I can try.

Reply:
@harens No, it does not. Permissions are set incorrectly then, and the directory is inaccessible. I do not know why, but guess just let it be how I have set it. I tried many options, this is what is confirmed to work.